server/server.go

```diff
@@ -426,6 +445,8 @@ func (s *Server) Run(ctx context.Context) error {
 	// nosemgrep: grpc-server-insecure-connection
 	grpcServer := grpc.NewServer(opts...)
 	openfgapb.RegisterOpenFGAServiceServer(grpcServer, s)
+	healthv1pb.RegisterHealthServer(grpcServer, s.healthManager.GetHealthServer())
+	reflection.Register(grpcServer)
```

Comment:
I added GRPC Server Reflection here to make it easier to discover the grpc endpoints as a grpc client.

See https://github.com/grpc/grpc/blob/master/doc/server-reflection.md. This is a really nice feature/integration to have for grpc and it made my testing of the functionality contained in this PR easier. That's why I added it.